src/GuppyDataExplorer/ExplorerTable/index.jsx

```diff
@@ -224,6 +226,45 @@ class ExplorerTable extends React.Component {
       });
     });
   };
+  /**
+   * Sets empty coumns visability to state showEmptyColumns
+   * @param colList: column configs for root table
+   * @returns column configs for root table with
+   * empty coumns visability set to state showEmptyColumns
+   */
+  markEmptyColums = (colList) => {
```

Comment:
should this just be inline rather then its own separate function? would like to know other peoples thoughts